apps/nextjs-website/src/_contents/translations.ts

```diff
@@ -107,10 +108,52 @@ export const translations = {
       ],
     },
     webinarBannerButtonContent: 'Scopri',
-    productsShowcase: {
+    ecosystem: {
       title: 'Scopri il nostro ecosistema',
-      products: [appIo, ioSign, pagoPa, send],
-    },
+      productsTabName: 'Per prodotti',
+      products: [
+        {
+          title: appIo.name,
```

Comment:
``Maybe it can be useful to use an util something like 
```
const productToEcosystemProduct = (product: Product) => ({
  title: product.name, text: product.description, href: product.slug, icon: product.logo.url
})
```
so that this array woul become something like 
```
products: [
  productToEcosystemProduct(appIo),
  productToEcosystemProduct(ioSign),
  productToEcosystemProduct(pagoPa),
  productToEcosystemProduct(send)
]
```